pkg/network/tracer/tracer_linux_test.go

```diff
@@ -1861,9 +1861,21 @@ func (s *TracerSuite) TestPreexistingConnectionDirection() {
 
 func (s *TracerSuite) TestPreexistingEmptyIncomingConnectionDirection() {
 	t := s.T()
+	t.Run("ringbuff_enabled", func(t *testing.T) {
+		// ringbuffers are enabled by default
+		c := testConfig()
+		testPreexistingEmptyIncomingConnectionDirection(t, c)
+	})
```

Comment:
As I see it - this test is going to run always, even if ringbuffer is not supported, which is in this case, we'll fallback to the original implementation, and then, there is no difference between running this to `ringbuff_disabled`
I think you should skip that test for unsupported kernels, or at least, add a log / comment, about the fallback mechanism

Reply:
fixed, thanks